source/installation/restore-minio.rst

```diff
@@ -39,17 +35,19 @@ and links to procedures for recovering from each:
      - MinIO detects when a node rejoins the deployment and begins proactively healing the node shortly after it is joined back to the cluster
        healing data previously stored on that node.
 
+Since MinIO can operate in a degraded state without significant performance
+loss, administrators can schedule hardware replacement in proportion to the rate
+of hardware failure. "Normal" failure rates (single drive or node failure) may
+allow for a more reasonable replacement timeframe, while "critical" failure
+rates (multiple drives or nodes) may require a faster response.
```

Comment:
```suggestion
rates (multiple drives or nodes) may require a faster response.

If a drive has partially failed or take a long time to complete single operations
it is recommended to unmount the drive. 
A missing drive will cause much less disruption than a drive that will block on
some operations.
```

Reply:
> takes a long time to complete a single operation

Is there a way for users to identify this as being caused by a single drive in particular?

I'm thinking to modify slightly as:

```
If a drive has partially failed or is showing increased errors (e.g. SMART), you can safely unmount the drive. Missing drives cause less disruption compared to a drives that are producing errors on read/write.
```

Thoughts @klauspost ?

Reply:
Timeouts/hanging calls are the most common sideeffect. SMART can give hints, but is just as hard to identify. Observing read/write times and watching out for timeouts in system logs can help this.